config.json

```diff
@@ -1 +1 @@
-{"current_lab": 1}
\ No newline at end of file
+{"current_lab": 2, "f1tenth_labs_openrepo_uri": "https://github.com/naman-gupta99/f1tenth_labs_openrepo.git"}
```

Comment:
I'm not sure about using your specific repo. Is there a reason for this?

Reply:
This is a mistake.